src/enumo/ruleset.rs

```diff
@@ -281,12 +258,19 @@ impl<L: SynthLanguage> Ruleset<L> {
         }
 
         let stats = json!({
-            "spec": name,
-            "num_rules": num_rules,
-            "num_baseline": baseline.len(),
-            "enumo_derives_oopsla": forwards_derivable,
-            "oopsla_derives_enumo": backwards_derivable,
-            "time": duration.as_secs(),
+            "domain": name,
```

Comment:
Instead of making the json fields baseline-specific, can they always be the same? That will simplify the to-html and to-latex scripts

Reply:
keys `[baseline_name, enumo_spec_name, num_rules, num_baseline, enumo_derives_baseline, baseline_derives_enumo, time, domain]` would be good I think